ta/pkcs11/src/pkcs11_attributes.c

```diff
@@ -553,12 +560,13 @@ static enum pkcs11_rc create_data_attributes(struct obj_attrs **out,
 }
 
 static enum pkcs11_rc create_pub_key_attributes(struct obj_attrs **out,
-						struct obj_attrs *temp)
+						struct obj_attrs *temp,
+						enum processing_func function)
 {
 	uint32_t const *mandated = NULL;
-	uint32_t const *opt_or_null = NULL;
+	uint32_t const *opt = NULL;
```

Comment:
keep original wording or shorten sufix to `_oon` (optional-or-null).
the reference is processed through `set_attributes_opt_or_null()` which differs from `set_optional_attributes()`. I need a second look whether optional or opt-or-null is expected for the attributes.

Reply:
Updated a bit. Please check it out.